instrumentation/instaawssdk/invoke_lambda.go

```diff
@@ -0,0 +1,47 @@
+package instaawssdk
+
+import (
+	"github.com/aws/aws-sdk-go/service/lambda"
+	otlog "github.com/opentracing/opentracing-go/log"
+
+	"github.com/aws/aws-sdk-go/aws/request"
+	instana "github.com/instana/go-sensor"
+	"github.com/opentracing/opentracing-go"
+)
+
+func StartInvokeLambdaSpan(req *request.Request, sensor *instana.Sensor) {
+	parent, ok := instana.SpanFromContext(req.Context())
+	if !ok {
+		return
+	}
+
+	sp := sensor.Tracer().StartSpan("aws.lambda.invoke",
+		opentracing.ChildOf(parent.Context()),
+	)
+
+	if ii, ok := req.Params.(*lambda.InvokeInput); ok {
+		if ii.FunctionName != nil {
+			sp.SetTag("function", *ii.FunctionName)
+		}
+
+		if ii.InvocationType != nil {
+			sp.SetTag("type", *ii.InvocationType)
+		}
+	}
+
+	req.SetContext(instana.ContextWithSpan(req.Context(), sp))
+	injectTraceContext(sp, req)
+}
+
+func FinalizeInvokeLambdaSpan(req *request.Request) {
+	sp, ok := instana.SpanFromContext(req.Context())
+	if !ok {
+		return
+	}
+	defer sp.Finish()
+
+	if req.Error != nil {
+		sp.LogFields(otlog.Error(req.Error))
```

Comment:
Same as above, let's just log it

Reply:
It is done in the same way in FinalizeS3Span, FinalizeSQSSpan, FinalizeDynamoDBSpan. Should we change it there as well?

Reply:
Good find 👍 I think we should not mark exit spans as errors in these cases too. But let's make it a separate PR 😃

Reply:
Alright, I probably got confused here. The error in this case is coming from the AWS side, so it's valid to mark the span as an error 👍 Please disregard my initial comment.